validator/client/src/main/java/tech/pegasys/artemis/validator/client/ValidatorClient.java

```diff
@@ -66,7 +65,7 @@ public ValidatorClient() {}
 
       if (selected_committees.size() > 0) {
         List<Integer> validators = selected_committees.get(0).getCommittee();
-        int shard = selected_committees.get(0).getShard().intValue();
+        int shard = (int) selected_committees.get(0).getShard();
```

Comment:
This is probably a good use case for `toIntExact` instead of typecasting. I know in theory this shouldn't overflow because the spec doesn't let it, but I'd like to know anyway.